applications/ConvectionDiffusionApplication/python_scripts/convection_diffusion_solver.py

```diff
@@ -472,7 +472,7 @@ def _assign_nodally_properties(self):
             for key, value in mat["Variables"].items():
                 var = KratosMultiphysics.KratosGlobals.GetVariable(key)
                 if (self._check_variable_to_set(var)):
-                    if value.IsDouble():
+                    if value.IsDouble() or value.IsInt():
```

Comment:
```suggestion
                    if value.IsNumber():
```

perhaps that makes more sense here

Reply:
I'd follow @philbucher's suggestion in here.